linphone.xcodeproj/project.pbxproj

```diff
@@ -1881,25 +1886,23 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				225F6C091B380E1B00A570DC /* libilbc.a in Frameworks */,
+				50F024BA1B910C6C000BC09F /* libxml2.2.dylib in Frameworks */,
 				225F6C081B380E0000A570DC /* libSKP_SILK_SDK.a in Frameworks */,
-				225F6C011B380DE600A570DC /* libxml2.a in Frameworks */,
 				C6F476531B27157D00F4C2A7 /* CoreLocation.framework in Frameworks */,
-				F0B026F31AA710AF00FF49F7 /* libiconv.dylib in Frameworks */,
```

Comment:
It seems like libiconv is being removed, and that libintl is being added. Not that there is anything wrong with doing this, it's just something I didn't have to do with my builds here.